integration-tests/tests/run-notebook.js

```diff
@@ -24,7 +24,10 @@ const testRunNotebookFn = _.flow(
   await click(page, clickable({ text: 'Create Notebook' }))
   await click(page, clickable({ textContains: notebookName }))
   await click(page, clickable({ text: 'Edit' }))
-  await select(page, 'ENVIRONMENT', 'Hail')
+  // wait for drawer to slide in before clicking, otherwise we'll miss the button
+  await delay(1000)
```

Comment:
I think our convention has typically been to use a findElement() that is on the page to ensure a thing is there rather than a delay, thoughts on finding something that's in the default drawer?

Reply:
`click()` does await the thing we ask it to click. Unfortunately, it exists in the DOM and is clickable while it is animating in. I figured this out by capturing a screenshot of the element, which resulted in an image of the correct size but with the wrong content; it only showed right-hand side of the button. From this, I conclude that puppeteer is finding the element, calculating the point on the screen that it's going to click (or the rectangle it's going to capture for the screenshot) and then clicking (or capturing the pixels). Since the panel is still sliding in, the element has moved far enough left that the click misses the button.

We theoretically had this same problem with the `select()` for Hail below. However, since that is a wide select box, puppeteer's chosen click point (presumably closer to the right edge of the panel than we might expect) still activated the select whereas it is more unreliable for a narrower element like a button.

Reply:
ooh okay that makes sense, is the 1 second based on the time we selected for the animation to take to slide in?

Reply:
Unfortunately not. The animation time is set to 0.2s (I think). I tried delaying for 500ms in the test but was still seeing the same failures.

Reply:
okay this sounds reasonable to me, thanks for the explanation!